yarn.lock

```diff
@@ -1114,6 +1114,11 @@
   resolved "https://registry.yarnpkg.com/@tsconfig/node16/-/node16-1.0.3.tgz#472eaab5f15c1ffdd7f8628bd4c4f753995ec79e"
   integrity sha512-yOlFc+7UtL/89t2ZhjPvvB/DeAr3r+Dq58IgzsFkOAvVC6NMJXmCGjbptdXdR9qsX7pKcTL+s87FtYREi2dEEQ==
 
+"@types/bluebird@*":
```

Comment:
damn these types are old

Reply:
Oh that's very strange, there was a release on npm 3 months ago https://www.npmjs.com/package/retry-as-promised

Reply:
Oh nevermind it's just not listed in github releases but it's still getting updated https://github.com/mickhansen/retry-as-promised/blob/master/package.json

Reply:
Actually I was wrong. They stopped managing github releases but have still been releasing source code changes. https://www.npmjs.com/package/retry-as-promised

Reply:
By the way this isn't a problem for this PR, our users won't be affected since we only use Options

Reply:
Yeah, it's a package made by one of the former sequelize maintainers. Might be good if we can get this project under the sequelize organisation to ensure the continuity better.

@mickhansen not sure how often you check this anymore, but what do you think?

Reply:
Thank you! :)

We can still merge this, renovate will handle opening the package update PR

Reply:
> Yeah, it's a package made by one of the former sequelize maintainers. Might be good if we can get this project under the sequelize organisation to ensure the continuity better.
> 
> @mickhansen not sure how often you check this anymore, but what do you think?

If this idea is approved, I think it's better that the repository created by other maintainers can also be moved into the organization, do you think that's better?

This is just to eliminate the possibility of the library becoming obsolete in the future if it's under one person, not that I expect that.

Reply:
> That would be the idea
> 
> But note that we can always fork if one of our dependencies becomes unmaintained, we did that recently with https://github.com/sequelize/toposort

Ah that's right

Reply:
I don't often check the Sequelize repository, but do try to keep on top of the packages directly under my GitHub user.
Looks like I forgot to actually publish the tags as releases on retry-as-promised.

Reply:
@esetnik https://github.com/mickhansen/retry-as-promised/releases/tag/v7.0.0